server/index.ts

```diff
@@ -28,6 +29,8 @@ headerSetup(app);
 app.use("/signup", signupRouter);
 app.use("/login", loginRouter);
 app.use("/", router);
+app.use('/api', router);
```

Comment:
delete please so you can handle the next comment

Reply:
Will do